controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -73,7 +73,7 @@ type StorageConsumerReconciler struct {
 	namespace       string
 }
 
-//+kubebuilder:rbac:groups=ocs.openshift.io,resources=storageconsumers,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=ocs.openshift.io,resources=storageconsumers,verbs=get;list;watch;create;update;patch
```

Comment:
We shouldn't remove the delete permission from here. It should be from ocs-provider-server rbac. Look for rbac/provider-role